heartbeat/monitors/mocks.go

```diff
@@ -22,42 +22,98 @@ import (
 	"regexp"
```

Comment:
This file is now not a `_test` file because it's used as a helper in other tasks elsewhere. I wish `_test` files could be used by `_test` files from other packages, but alas it's not possible (for some good reasons)